tunnel/tunnel.go

```diff
@@ -607,7 +612,12 @@ func buildSSHChannels(serverName, channelType string, source, destination []stri
 func getForward(channelType, serverName string) (*ForwardConfig, error) {
 	var f *ForwardConfig
 
-	cfg, err := NewSSHConfigFile()
+	cfgPath, err := NewDefaultSSHConfigFilePath()
```

Comment:
This line is ignoring the possibility of the user providing a non-default ssh configuration file path.

Reply:
Now I see, passed the config parameter.